cluster/manifests/spot-node-rescheduler/cronjob.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: node-spot-rescheduler
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: node-spot-rescheduler
+spec:
+  schedule: "*/5 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 1800
+      backoffLimit: 1
+      template:
+        metadata:
+          labels:
+            application: kubernetes
+            component: node-spot-rescheduler
+        spec:
+          serviceAccountName: node-spot-rescheduler
+          restartPolicy: Never
+          containers:
+          - name: node-spot-rescheduler
+            image: container-registry-test.zalando.net/teapot/spot-node-rescheduler-test:pr-1-4
+            env:
+            - name: EXPERIMENTAL_SPOT_RESCHEDULER
+              value: "{{ .ConfigItems.experimental_spot_rescheduler }}"
```

Comment:
I think better than having this as a flag we can just have and if around this whole manifest. Like we do here: https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/kube-downscaler/deployment.yaml#L1


And the corresponding config in the `deletions.yaml` to be able to drop it when the config-item is disabled: https://github.com/zalando-incubator/kubernetes-on-aws/blob/c5823ca2d00d73c4bb2cc5b365cf80abf4704d42/cluster/manifests/deletions.yaml#L89-L103